config/redirects.yml

```diff
@@ -337,7 +337,7 @@ redirects:
   "/urgent-call-for-qualified-teachers": "/returning-to-teaching"
 
   # Blog pages
-  "/blog/5-reasons-to-attend-a-train-to-teach-event": "/blog/5-reasons-to-attend-a-get-into-teaching-event"
```

Comment:
I think we need to leave this redirect in but just change the destination, in case there are any old links to it ie

"/blog/5-reasons-to-attend-a-train-to-teach-event": "/events/about-get-into-teaching-events"

(We only renamed the blog last year when the event names changed)